scss/civicrm/common/_tables.scss

```diff
@@ -69,6 +69,10 @@ table {
     > tr:first-child > th {
       border-color: $crm-grayblue-dark;
     }
+
+    div.sticky-header {
```

Comment:
always need to ask: do we really need to specify the tag here?

Reply:
Yes, it's to handle specificity of core css file which has same structure.